src/main/twirl/views/DataLandingPage.scala.html

```diff
@@ -73,6 +73,9 @@
 				@if(uploadComplete) {
 					@accessUrlRow(dobj)
 				}
+				@for(keywords <- dobj.keywords){
+					@PanelRow("Keywords", keywords.mkString(", "))
```

Comment:
Should the keywords be sorted? Can be done here or in `DataObject.keywords` method.